### ISSUE TITLE
quarkus.servlet.context-path is not proposed as completion item 

### ISSUE THREAD

**Issue by fbricon** [redhat-developer/quarkus-ls#39]
New in quarkus 0.20.0 : 

> By default Undertow will serve content from under the root context. If you want to change this you can use the quarkus.servlet.context-path config key to set the context path.

See https://github.com/quarkusio/quarkus/commit/75040ec51cfa525cd9f3b6fdc0946dfd9833da8b#diff-df48894363b7b7f15b5e5e5db756699fR9-R17

I added this to the dependencies:

```xml
<dependency>
  <groupId>io.quarkus</groupId>
  <artifactId>quarkus-undertow</artifactId>
</dependency>
```

but don't get `quarkus.servlet.context-path` proposed. Either a bug or might be the undertow deployment classes are not available to the project's classpath.

**Comment by fbricon** [comment]
So, it looks like we need to build a runtime classpath from the quarkus-extensions/META-INF/quarkus-extension.properties deployment artifacts.

For quarkus-undertow, the quarkus-extension.properties looks like:
```
#Generated by extension-descriptor
#Thu Aug 15 13:57:50 CEST 2019
deployment-artifact=io.quarkus\:quarkus-undertow-deployment\:0.21.1
```

**Comment by fbricon** [comment]
Again, it'd would be much easier if the config metadata was embedded in the extension jars directly. cf. https://github.com/quarkusio/quarkus/issues/1655

**Comment by angelozerr** [comment]
@fbricon it seems ServletConfig belong sto `quarkus-undertow-deployment` articfact. So I suppose if you add in your pom.xml this dependency, it should work, no?

```xml
<dependency>
  <groupId>io.quarkus</groupId>
  <artifactId>quarkus-undertow-deployment</artifactId>
</dependency>
```

**Comment by fbricon** [comment]
yes adding `quarkus-undertow-deployment` to the pom.xml works, but we should be able to avoid that, ideally, this is a quarkus runtime dependency only
